plugin/table/table_test.go

```diff
@@ -17,7 +17,9 @@ func TestGoldenFiles(t *testing.T) {
 			converter.WithPlugins(
 				base.NewBasePlugin(),
 				commonmark.NewCommonmarkPlugin(),
-				NewTablePlugin(),
+				NewTablePlugin(
+					WithCellPadding(CellPaddingAligned),
```

Comment:
Please remove that since we want to test the default here. 

Instead we can set a default:
```go
func NewTablePlugin(opts ...option) converter.Plugin {
	plugin := &tablePlugin{
		cellPaddingBehavior: CellPaddingAligned,
	}
```